xds/src/test/java/io/grpc/xds/XdsServerBuilderTest.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2019 The gRPC Authors
```

Comment:
```suggestion
 * Copyright 2020 The gRPC Authors
```

Reply:
The file was moved from under sds to this new location and a test was added. Not sure if the year needs to be updated